docs/zh-CN/administrator-guide/operation/tablet-meta-tool.md

```diff
@@ -136,4 +136,13 @@ api：
 ./lib/meta_tool --operation=show_meta --root_path=/path/to/root_path --pb_header_path=path
 ```
 
+### 展示 pb 格式的 Segment Footer
+
+这个命令是为了查看SegmentV2 的SegmentFooterPB信息，以json 形式展示出来
```

Comment:
Would better show the command result by a simple example. Because most of people don't know what's SegmentFooterPB.